rebar.config.script

```diff
@@ -162,18 +162,18 @@ DepDescs = [
 {mochiweb,         "mochiweb",         {tag, "v2.20.0"}},
 {meck,             "meck",             {tag, "0.8.8"}},
 {recon,            "recon",            {tag, "2.5.0"}}
-],
+].
 
-WithProper = lists:keyfind(with_proper, 1, CouchConfig) == {with_proper, true},
+WithProper = lists:keyfind(with_proper, 1, CouchConfig) == {with_proper, true}.
 
 OptionalDeps = case WithProper of
     true ->
         [{proper, {url, "https://github.com/proper-testing/proper"}, {tag, "v1.3"}}];
     false ->
         []
-end,
+end.
```

Comment:
Why did you change all of the statement separators?

Reply:
Because I burnt a day on trying to understand why I had a bug that jumped around when I changed `,` to `.`. Bob agreed we should pick one and stick to it. It's unrelated, I can move to a separate commit, but I don't see why we shouldn't do it.

Reply:
Ah, that's more or less what i assumed. I'm fine not breaking it out into a separate commit.